.github/workflows/release.yml

```diff
@@ -54,39 +52,21 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
 
-    services:
-      postgres:
-        image: postgres:10
-        env:
-          POSTGRES_DB: test_aiida
-          POSTGRES_PASSWORD: ''
-          POSTGRES_HOST_AUTH_METHOD: trust
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-        - 5432:5432
-      rabbitmq:
-        image: rabbitmq:3.8.14-management
-        ports:
-        - 5672:5672
-        - 15672:15672
-
```

Comment:
> Last PR related to CI (promise!), I saved the perhaps controversial change for last. :-)

Not sure which part of the PR you thought was controversial, I am betting the auto-requirements update, but I think sneaking this change in might be a better candidate. I would be hesitant to change the release workflow since we only just started using the presto suite. I would wait until we have more confidence in its reliability and utility

Reply:
Haha, fair enough, I've reverted the change.
(and yes, it was the dependabot config :-) )